stumpy/Mmotifs.py

```diff
@@ -0,0 +1,297 @@
+import numpy as np
+import logging
+import stumpy
+import matplotlib.pyplot as plt
+from . import core
+from . import config
+from stumpy.core import _preprocess
+
+logger = logging.getLogger(__name__)
+
+
+def Mmotifs(

```

Comment:
Let's use `mmotifs` to be consistent with STUMPY's other function names that all use lowercase